src/gsi/read_atms.f90

```diff
@@ -195,7 +195,7 @@ subroutine read_atms(mype,val_tovs,ithin,isfcalc,&
   real(r_kind)    :: ptime,timeinflat,crit0,score
   integer(i_kind) :: ithin_time,n_tbin
   integer(i_kind),pointer :: it_mesh => null()
-  integer(kind=4) :: good,bin,bin2,Obindx,maxPerBin,numBinsWithObs
+  integer(kind=4) :: good,bin,bin2,Obindx,maxPerBin,numBinsWithObs,numobs
```

Comment:
Didn't notice this before.  We prefer to **not** use a number to specify the variable `kind`.  

`src/gsi/kinds.F90` has parameter
```
i_long    - specification kind for long (4-byte) integer variable
```
Can we replace `integer(kind=4)` with `integer(kind=i_long)`?  Doing so requires adding `i_long` to the `use kinds` statement on line 75.